### PR TITLE
Added the Japanese translation for homepage. (Re PR)

### PR DESCRIPTION
**Pull request by Hikai**
Added the Japanese translation for homepage. again PR.

I made a mistake and did it again. I'm sorry.
(Rebase is too hard.)

Thank you for @saneyuki for review!

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo.org/60)

### CODE REVIEW

File: ja-JP/index.html [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+
+      </div>
+    </div>
+
+    <div class='row'>
+      <div class='col-sm-12'>
+        <h2 class='header'>開発に参加する</h2>
+
+        <p id='contributing'>
+          Servoプロジェクトは、経験が豊富な開発者から初心者まで、多くの皆さんによる開発への参加を受け入れます。
```
Comment (tetsuharuohzeki) [inline ja-JP/index.html]: > 多くの皆さんによる開発への参加を受け入れます。

多くの皆さんによる開発への参加を歓迎しています。

would be more nice.

File: ja-JP/index.html [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+
+          Servoをさらに良いものにする為に、全世界で数百人の開発者が一緒に開発に取り組んでいます。
+          この取り組みに参加するにはページの末尾に書かれたリンクを参照してください。
```
Comment (tetsuharuohzeki) [inline ja-JP/index.html]: > ページの末尾に書かれたリンクを

ページの末尾に記載されたリンクを

File: ja-JP/index.html [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+      </div>
+    </div>
+
+    <div class='row'>
+      <div class='col-sm-12'>
+        <h2 class='header'>行動規範</h2>
+
+        <p>
+          私たちは
+          <a href="https://www.rust-lang.org/conduct.html">Rust 行動規範</a>に
```
Comment (tetsuharuohzeki) [inline ja-JP/index.html]: > Rust 行動規範

Rustの行動規範

File: ja-JP/index.html [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+    </div>
+
+    <div class='row'>
+      <div class='col-sm-12'>
+        <h2 class='header'>コミュニケーション</h2>
+
+        <p>
+          Servoの開発者の間では
+          <a href="https://wiki.mozilla.org/IRC">irc.mozilla.org</a>の#servoチャンネルが活発に使用されています。
+          また、<a href="https://lists.mozilla.org/listinfo/dev-servo">開発メールリグリスト</a>でも議論が行われており、自由に参加できます。
```
Comment (tetsuharuohzeki) [inline ja-JP/index.html]: > 開発メールリグリスト

開発メーリングリスト

File: ja-JP/index.html [hunk 5]
```diff
@@ -0,0 +1,256 @@
...
+        </p>
+
+        <p>
+          何か問題を解決したい場合、まずGitHub Issueに問題・意見を登録してください。
+          これにより同じ問題に対しての重複した努力を避けることができます。
+        </p>
+
+        <p>
+          もし簡単なバグを探しているのならGitHubの
+          <a href="https://github.com/servo/servo/labels/E-easy">E-Easy issue タグ</a>を
```
Comment (tetsuharuohzeki) [inline ja-JP/index.html]: > E-Easy issue タグ

E-Easy ラベル

File: ja-JP/index.html [hunk 6]
```diff
@@ -0,0 +1,256 @@
...
+        <h3>ビルド</h3>
+        <p>
+          ServoはRustのパッケージマネージャーであるCargoでビルドされています。
+          また、MozillaのMachツールチェインがビルドやその他のタスクのインターフェースとして用いられています。
+        </p>
+
+        <h4>一般的なビルド方法</h4>
+        <p>
+          Servoを開発モードでビルドすると開発に役立ちます。
+          しかし、生成されたバイナリの動作は非常に遅いです。
```
Comment (tetsuharuohzeki) [inline ja-JP/index.html]: > 非常に遅いです

非常に低速なことに注意してください。

would be more nice.

**Review by tetsuharuohzeki — CHANGES_REQUESTED** [review]
@hikai  Thank you for your response!

I reviewed your updated patch. 

- I find more nice phrase at some points. They are my mistaken review in https://github.com/servo/servo.org/pull/58. I'm sorry.
- I find some miss-spells. Please fix them.

**Comment by Hikai** [comment]
Sorry, I late to commit because so busy.
Is this the right way to do a commit squash?

And Thank you for your review!

**Comment by jdm** [comment]
Looks great. Thanks!

**Review by tetsuharuohzeki — APPROVED** [review]

**Comment by tetsuharuohzeki** [comment]
I feel this translation is good.

-------------------

@hikai 

> Sorry, I late to commit because so busy.
> Is this the right way to do a commit squash?

No problem!
Yes, squash is the right wary. Thank YOU!

-------------------

@jdm 

Could you review this pull request for the technical aspect?
I can review the translation feeling. But I'm not sure about this repository.

**Comment by jdm** [comment]
Thanks for reviewing the language changes @saneyuki!

**Pull request merged**